static/js/reactions.js

```diff
@@ -0,0 +1,176 @@
+var reactions = (function () {
+var exports = {};
+
+function send_reaction_ajax(message_id, emoji_name, operation) {
+    var args = {
+        url: '/json/messages/' + message_id + '/emoji_reactions/' + emoji_name,
+        data: {},
+        success: function () {},
+        error: function (xhr) {
+          var response = channel.xhr_error_message("Error sending reaction", xhr);
+          blueslip.error(response);
+        }
```

Comment:
This isn't indented properly and throws an exception if you try a nonexisting emoji.